frontend/src/pages/Testbed.tsx

```diff
@@ -61,14 +63,81 @@ const logChange = (...args: unknown[]) => {
 
 /** test and example usage of formatting, elements, components, etc. */
 const TestbedPage = () => {
+  /** generate fake node data */
+  const nodes = Array(200)
+    .fill(null)
+    .map(() => ({
+      id: uniqueId(),
+      label: sample([
+        "Lbl.",
+        "Label",
+        "Long Label",
+        "Really Long Label",
+        undefined,
+      ]),
+      type: sample([
+        "gene",
```

Comment:
@vincerubinetti at what stage do these attributes need to be defined? e.g., circles for genes, rounded-edge-rectangles for diseases, triangles for drugs, etc?

Also, right now we only have protein domains. This may change down the line, though (or when adapted to other projects).

Reply:
What stage as in what stage of our development process? Or stage as in, flow of data through the frontend?

Right now, I've designed this as a generic component. That is, not too hard-coded to the exact specific use-case you might have in mind right now. This could allow reuse in a different project, or in this project if you ever want to use the network viz for some different data.